src/cli/self_update/test.rs

```diff
@@ -37,37 +42,35 @@ pub fn with_saved_path(f: &mut dyn FnMut()) {
 }
 
 #[cfg(windows)]
-pub fn get_path() -> io::Result<Option<RegValue>> {
-    get_reg_value(&RegKey::predef(HKEY_CURRENT_USER), "Environment", "PATH")
+pub fn get_path() -> io::Result<Option<String>> {
+    get_reg_string(CURRENT_USER, "Environment", "PATH")
 }
 
 #[cfg(windows)]
-pub fn with_saved_reg_value(root: &RegKey, subkey: &str, name: &str, f: &mut dyn FnMut()) {
+pub fn with_saved_reg_string(root: &Key, subkey: &str, name: &str, f: &mut dyn FnMut()) {
```

Comment:
As @rami3l suggested, this renaming isn't really necessary for porting to windows-registry. As such, it should be in a separate commit. (If you're renaming a family of related functions those can probably all go in the same commit.)